docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,130 @@
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## Who we're looking for
+
+We're looking for someone with strong core JavaScript skills. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Required
+
+- **Expertise in the modern JavaScript ecosystem**.
+  Pigment CSS is built on the shoulders of giants, you need to use technologies such as TypeScript, Node.js, Babel, Rust, or SWC.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping beautiful user interfaces and intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**.
+  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**.
+  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Leadership skills**.
+  You will help build the community and collaborate with it.
+  You will guide and mentor contributors, steering the community in the right direction.
+- **Past work with front-end infrastructure**.
+  Perhaps you've written a Babel plugin or even worked on a problem similar to the ones Pigment CSS aims to address.
+  It would be great if you could address this in your cover letter!
+
+### Nice to have (but not required)
+
+- **You've maintained an active repository before**.
+  Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams.
+  Previous experience with highly active repository workflows is a definite plus for this role.
+- **You have contributed code to a CSS-in-JS library**. A history of contributing to a CSS-in-JS library is a plus.
```

Comment:
```suggestion
- **You've contributed to a CSS-in-JS library** in the past.
```

Cutting the "CSS-in-JS library" repetition. No need to say it's a plus here, too; it's already said in the "nice to have" title—can be simpler.